pkg/controller/kmeshsecurity/option.go

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kmeshsecurity
+
+import (
+	"time"
+
+	"istio.io/istio/pkg/env"
+	"istio.io/istio/pkg/security"
+)
+
+var 	(
+	CACertFilePath = ""
+	PilotCertProvider = env.Register("PILOT_CERT_PROVIDER", "istiod",
+	"The provider of Pilot DNS certificate.").Get()
+	CSRSignAddress = env.Register("MESH_CONTROLLER", "istiod.istio-system.svc:15012", "").Get()
```

Comment:
these params should be passed in, we donot want to make flags distributed everywhere across packages